components/Image.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { ReactElement } from 'react';
 import NextImage from 'next/image';
-import { Avatar, createStyles, makeStyles, Theme } from '@material-ui/core';
+import { Avatar, Theme } from '@mui/material';
+import createStyles from '@mui/styles/createStyles';
```

Comment:
Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```